pom.xml

```diff
@@ -158,10 +158,108 @@ The MIT License (MIT)
           <plugin>
             <groupId>com.qulice</groupId>
             <artifactId>qulice-maven-plugin</artifactId>
-            <version>0.17.3</version>
+            <version>0.18.19</version>
             <configuration>
               <excludes combine.children="append">
                 <exclude>checkstyle:/src/site/resources/.*</exclude>
+                <!--
+                 @todo #1151:30min To fix qulice build erros is necessary update it.
```

Comment:
@fabriciofx Is necessary to update qulice again? Or updating from 0.17.3 to 0.18.19 already fixed the problems? Puzzle text is not clear

Reply:
@paulodamaso I'll fix it.